extensions/wikia/Sitemap/maintenance.php

```diff
@@ -34,17 +34,6 @@
 	$indexes[ $namespace ] = $sitemap->cachePages( $namespace );
 	echo " pages " . count( $indexes[ $namespace ] );
 	echo " done\n";
-
-	$includeVideo = (bool) F::app()->wg->EnableVideoSitemaps;
```

Comment:
This variable has no default in the config and is set on only 14 wikis - in all case to `false`.